src/chttpd/src/chttpd_misc.erl

```diff
@@ -50,6 +50,7 @@ handle_welcome_req(#httpd{method='GET'}=Req, WelcomeMessage) ->
         {couchdb, WelcomeMessage},
         {version, list_to_binary(couch_server:get_version())},
         {git_sha, list_to_binary(couch_server:get_git_sha())},
+        {uuid, list_to_binary(couch_server:get_uuid())},
```

Comment:
`couch_server:get_uuid/0` returns binary (https://github.com/apache/couchdb/blob/master/src/couch/src/couch_server.erl#L68 `?l2b` macro stands for `list to binary`). Attempt to call `list_to_binary` on a binary argument causing `badarg` exception.